### ISSUE TITLE
Take an action to dispatch other actions shorthand

### ISSUE THREAD

**Issue by TecHunter** [redux-saga/redux-saga#1835]
Hello,

I would like to know if there is a way to redirect an action to another(s)?
For now I did this:

```
export const takePut = function takePut(taken, forwardTo) {
  return takeEvery(taken, function* forward(action) {
    yield all(map(forwardTo, type => put({
      ...action,
      type
    })));
  });
};
```

Exemple here: https://codesandbox.io/s/jlznqqlnqy

Taking action INCREMENT to put actions INCREMENT_100 and INCREMENT_10000. It's a shortcut for take/put

**Comment by shinima** [comment]
@TecHunter It is okay, just go ahead.

**Comment by TecHunter** [comment]
it's okay? I don't understand your answer
